src/houghtransform.jl

```diff
@@ -81,30 +90,29 @@ function hough_transform_standard(
 
     #Finding local maximum lines
     validLines = Vector{CartesianIndex{2}}(undef, 0)
-    findlocalmaxima!(validLines, accumulator_matrix, threshold)
+    findlocalmaxima!(validLines, accumulator_matrix, vote_threshold)
 
     #Sorting by value in accumulator_matrix
     @noinline sort_by_votes(validLines, accumulator_matrix) = sort!(validLines, lt = (a,b)-> accumulator_matrix[a]>accumulator_matrix[b])
     sort_by_votes(validLines, accumulator_matrix)
 
-    linesMax = min(linesMax, length(validLines))
+    max_linecount = min(max_linecount, length(validLines))
 
     lines = Vector{Tuple{Float64,Float64}}(undef, 0)
 
-    #Getting lines with Maximum value in accumulator_matrix && size(lines) < linesMax
-    for l in 1:linesMax
+    #Getting lines with Maximum value in accumulator_matrix && size(lines) < max_linecount
+    for l in 1:max_linecount
         lrho = ((validLines[l][2]-1) - (numrho-1)*0.5)*ρ
         langle = θ[validLines[l][1]-1]
         push!(lines,(lrho,langle))
     end
 
     lines
-
 end
 
 """
 ```
-circle_centers, circle_radius = hough_circle_gradient(img_edges, img_phase, scale, min_dist, vote_thres, min_radius:max_radius)  
+circle_centers, circle_radius = hough_circle_gradient(img_edges, img_phase, radii; scale=1, min_dist=minimum(radii), vote_threshold=minimum(radii))
```

Comment:
Should the default for `vote_threshold` also depend on `scale`?

Reply:
Maybe, but I wanted to keep things simple stupid. The situation is this:
* A perfect circle or radius r will vote r*pi times for its center. It will do this regardless of the scale.
* A real world circle spreads the votes over a neighborhood of the true center
* Depending on the scale parameter, this neighborhood may be conflated to a single point
* If `length(radii) > 1` the circle does multiple voting rounds and votes from nearby radii will accidentally hit the center also. Especially if scale is large.

So a more complicated heuristic would be something like 
```const * minimum(radii) * min(scale, length(radii))```
Would you prefer that? Also is the algorithm in the package published somewhere?

Reply:
Thanks for the reference (unfortunatelly I cannot access the paper though)!
I like the idea of providing the vote threshold as a fraction of full circle votes very much.
The api could be to pass `vote_threshold = FullCircleVotes(0.4)`.

I think you are right that in an ideal situation `minimum(radii)*min(scale, length(radii))` corresponds to `1/(2pi)` fraction of votes. But there are complications:
* The dependence of full circle votes on scale etc is very spiky
* Larger circles have more votes then smaller ones
* If the true radius becomes extremal `r_true in extrema(radii)` there are only half as much nearby radius votes, as for `r_true` in the middle of `radii`.

At least the first problem could be overcome by actually counting the votes for a perfect circle:
```julia
function compute_vote_threshold(obj::FullCircleVotes, img, scale, radii...)
     # select some r in the middle of radii
     # create perfect circle of radius r
     # count its vote for its true center
end
```